test/gtest/uct/test_pending.cc

```diff
@@ -473,11 +473,8 @@ UCS_TEST_SKIP_COND_P(test_uct_pending, pending_fairness, RUNNING_ON_VALGRIND)
     initialize();
     check_caps(UCT_IFACE_FLAG_AM_SHORT | UCT_IFACE_FLAG_PENDING);
     if (m_e1->iface_attr().cap.flags & UCT_IFACE_FLAG_CONNECT_TO_IFACE) {
-        N = 128;
-
-        if (has_transport("tcp")) {
-            N = ucs_min(N, max_connect_batch());
-        }
+        N = has_transport("tcp") ?
+            ucs_min(128, max_connect_batch()) : 128;
```

Comment:
you have ` has_transport("tcp")` check inside `max_connect_batch()`

Reply:
good catch! done